tests/test_backend.py

```diff
@@ -1726,6 +1742,26 @@ def test_isnan(self):
             self._run_test_case([_OUTPUT], {_INPUT: x_val})
             tf.reset_default_graph()
 
+    @staticmethod
+    def _has_specific_node(graph, checker):
+
+        ops = graph.get_nodes()
+        for node in ops:
+            if checker(graph, node):
+                return True
+        # doesn't find anything
+        return False
+
+    @staticmethod
+    def _is_upsample_with_const_input(graph, node, const_val):
```

Comment:
how about something like ```validate_const_node(node, value)```
put it in tests/common.py, so other test scripts can use it.